CMakeLists.txt

```diff
@@ -208,6 +208,11 @@ zephyr_compile_options(
   ${TOOLCHAIN_C_FLAGS}
 )
 
+# @Intent: Enforce standard type correspondance to match Zephyr usage.
+zephyr_compile_options(
```

Comment:
Sorry, but now that this commit: "4ddbc0096a5e9925a27ee3a0b130fd78373650b6"

is merged, we must, for toolchain portability reasons, use

toolchain_cc_imacros

instead of

zephyr_compile_options

and patch this line:

https://github.com/zephyrproject-rtos/zephyr/commit/4ddbc0096a5e9925a27ee3a0b130fd78373650b6#diff-6c754805e58aa2a9bce0dbdf0414d7f5R7

to use the format that does not have whitespace, meaning "--imacros=foo" instead of "-imacros foo".